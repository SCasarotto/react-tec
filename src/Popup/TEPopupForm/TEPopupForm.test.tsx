import React from 'react'
import ReactDOM from 'react-dom'
import { TEPopupForm } from '../..'

describe('TEPopupForm', () => {
	it('renders without crashing', () => {
		const div = document.createElement('div')
		ReactDOM.render(<TEPopupForm visible onClose={() => {}} />, div)
		ReactDOM.unmountComponentAtNode(div)
	})
})
