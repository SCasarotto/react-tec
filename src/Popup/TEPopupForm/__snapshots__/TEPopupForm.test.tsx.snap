// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TEPopupForm matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="sc-gtssRu ZemqJ TEPopup sc-eCApGN jrzxTO TEPopupForm "
  >
    <div
      class="sc-dlnjPT goRCKu TEPopupContent"
    >
      <form
        class="sc-bdnylx gCsuFE TEForm TEPopupFormForm"
      >
        <div
          class="sc-jSFkmK eXNlxJ TEPopupFormContent"
        />
        <div
          class="sc-gKAblj hdBIBd TEPopupFormButtonContainer"
        >
          <button
            class="sc-hKFyIo ktDqPm TEButton sc-iCoHVE evUvse TEPopupFormButton TEPopupFormLeftButton"
            type="button"
          >
            Cancel
          </button>
          <button
            class="sc-hKFyIo ktDqPm TEButton sc-fujyUd jOdStx TEPopupFormButton TEPopupFormRightButton"
            type="submit"
          >
            Submit
          </button>
        </div>
      </form>
    </div>
  </div>
</DocumentFragment>
`;
